_Classes/DeusEx/Classes/DeusExWeapon.uc

```diff
@@ -53,6 +53,9 @@ enum ELockMode
 var bool				bReadyToFire;			// true if our bullets are loaded, etc.
 var() int				LowAmmoWaterMark;		// critical low ammo count
 var travel int			ClipCount;				// number of bullets remaining in current clip
+var travel int			ARClipSize;
```

Comment:
Is it possible to move all the GL specific code to WeaponAssaultGun.uc?

Reply:
Not possible because the rifle needs to call the script inside LoadAmmo(), unless polymorphism is a thing here - which it isn't afaik